python_modules/dagster/dagster/__init__.py

```diff
@@ -454,6 +454,10 @@
 from dagster._core.instance_for_test import instance_for_test as instance_for_test
 from dagster._core.launcher.default_run_launcher import DefaultRunLauncher as DefaultRunLauncher
 from dagster._core.log_manager import DagsterLogManager as DagsterLogManager
+from dagster._core.run_coordinator.queued_run_coordinator import (
+    QueuedRunCoordinator as QueuedRunCoordinator,
+    SubmitRunContext as SubmitRunContext,
+)
```

Comment:
some concern on import cost if there are any heavy things contained within. Theres a test for specific known heavy libs so that should prevent anything too egregious

Reply:
Yeah that test was failing until I made some minor changes